### DISCUSSION TITLE
wiki evolutions is not specific enough

### DISCUSSION THREAD

**Discussion by tails1154** [haven1433/HexManiacAdvance#202] [Q&A] [answered]
So, I was trying to make an evolution for the Pokémon from Pokémon violet called Sprigatito, and I used the ? to ?~25, So I was following the tutorial here: https://github.com/haven1433/HexManiacAdvance/wiki/Evolution-Editing | It was stopping the evolution in the middle of it ongoing and It told me how to fix it but I have no idea what It is talking about add “special 0x16F” It does not tell me how or where to put it. any help would be appreciated. Have a nice day!

**Comment by tails1154** [comment]
I just put this in a script

**Comment by haven1433** [comment, marked as answer]
I updated the documentation, thanks for the heads up

**Reply by tails1154** [reply]
okay!
